bcos-gateway/bcos-gateway/libp2p/P2PMessage.h

```diff
@@ -123,9 +123,15 @@ class P2PMessage : public Message, public bcos::ObjectCounter<P2PMessage>
     using Ptr = std::shared_ptr<P2PMessage>;
 
     /// length(4) + version(2) + packetType(2) + seq(4) + ext(2)
-    const static size_t MESSAGE_HEADER_LENGTH = 14;
-    const static size_t MAX_MESSAGE_LENGTH =
+    constexpr static size_t MESSAGE_HEADER_LENGTH = 14;
+    constexpr static size_t MAX_MESSAGE_LENGTH =
         100 * 1024 * 1024;  ///< The maximum length of data is 100M.
+
+    /// For RSA public key, the prefix length is 18 in hex, used for print log graciously
+    constexpr static size_t RSA_PUBLIC_KEY_PREFIX = 18;
```

Comment:
Maybe you could add a comment to explain why this prefix will happen. 
because at first, this is not raw public key, it is the der format of public key